### PR TITLE
chore: add additionalData atribute in Extra interface

### PR DESCRIPTION
**Pull request by MilkaCab**

### CODE REVIEW

File: src/interfaces/index.ts [hunk 1]
```diff
@@ -196,3 +197,8 @@ export type SoapClientConfig = SoapBaseClientConfig & ExtendedConfig;
 export type RestClientConfig = RestBaseClientConfig & ExtendedConfig;
 
 export type ClientConfig = SoapClientConfig | RestClientConfig;
+
+export interface ServiceIdExtended {
+  serviceId: string;
```
Comment (rcabralyu) [inline src/interfaces/index.ts:202]: add requestId and change serviceId as optional

File: src/generator/index.ts [hunk 2]
```diff
@@ -69,3 +72,7 @@ export class MonoClientGenerator {
     }
   }
 }
+
+function isServiceIdExtended(object: any): object is ServiceIdExtended {
+  return typeof object != 'string' && typeof object != 'number' && 'serviceId' in object;
```
Comment (rcabralyu) [inline src/generator/index.ts:77]: If you change serviceId as optional you can do this
```suggestion
  return typeof object === 'object' && object != null;
```

Comment (MilkaCab) [reply src/generator/index.ts]: done!

File: src/generator/index.ts [hunk 3]
```diff
@@ -27,16 +28,27 @@ export class MonoClientGenerator {
     serviceId?: string | number,
     requestId?: string | number
   ): MonoClient<RestClientConfig, RestRequest>;
+  get(
+    params: SoapBaseClientConfig,
+    serviceId?: ServiceIdExtended,
+    requestId?: string | number
```
Comment (rcabralyu) [inline src/generator/index.ts:34]: Remove this line

Comment (MilkaCab) [reply src/generator/index.ts]: done!

**Review by rcabralyu — APPROVED** [review]

**Pull request merged**